protocol/scripts/genesis/prod_pregenesis.sh

```diff
@@ -107,6 +107,7 @@ function overwrite_genesis_production() {
 	dasel put -t string -f "$GENESIS" '.app_state.gov.params.quorum' -v '0.33400' # 33.4%
 	dasel put -t string -f "$GENESIS" '.app_state.gov.params.threshold' -v '0.50000' # 50%
 	dasel put -t string -f "$GENESIS" '.app_state.gov.params.veto_threshold' -v '0.33400' # 33.4%
+	dasel put -t string -f "$GENESIS" '.app_state.gov.params.min_initial_deposit_ratio' -v '0.20000' # 20%
```

Comment:
should we modify testnet genesis script (`testing/testnet-external/pregenesis.sh`) too to have this value?

Reply:
good callout, done!